shared/flexible-classroom/quickstart/ios.mdx

```diff
@@ -65,7 +65,7 @@ Follow these steps to configure and launch your <Vg k="FC" /> from the downloade
 1. Navigate to the `CloudClass-iOS/App` directory:
 
    ```bash
-   cd CloudClass-iOS/App
+   cd flexible-classroom-ios/App
    ```
 
 1. Install project dependencies using the following command:
```

Comment:
Mention of the old repo below (After installation, open the `CloudClass-iOS` folder in the Finder window,...)